torch/nn/modules/loss.py

```diff
@@ -1246,6 +1251,121 @@ def forward(self, anchor: Tensor, positive: Tensor, negative: Tensor) -> Tensor:
                                      eps=self.eps, swap=self.swap, reduction=self.reduction)
 
 
+class TripletMarginWithDistanceLoss(_Loss):
+    r"""Creates a criterion that measures the triplet loss given input
+    tensors :math:`a`, :math:`p`, and :math:`n` (representing anchor,
+    positive, and negative examples, respectively); and a nonnegative,
```

Comment:
semicolon here doesn't seem correct

Reply:
Were you thinking a comma instead, or no punctuation?  I was trying to group {tensors a, p, n} and {distance function} since there are several commas within each, but since the parenthetical statement after the tensor can serve as a separator, I can see it working in other ways.

Reply:
Comma seems fine.

Reply:
Fixed!